odoo/fields.py

```diff
@@ -1979,27 +1980,30 @@ def _convert(self, value, record, validate):
             'strip_classes': self.strip_classes
         }
 
+        # TODO: Wouldn't it be better to move all the normalize stuff here
```

Comment:
I meant before the `return value` a few lines above, so all call to `_convert()` does it.
And so even when saved by the admin, the normalize part will be done.
Isn't it strange to have this conditioned? Having that based on right or on field attribute (sanitize) doesn't seem right?
Eg, why would we remove the wrapping `<div/>` element only on sanitized fields?